README.md

```diff
@@ -145,6 +145,7 @@ Here is a partial list of open source projects that have adopted `sse2neon` for
 ## Reference
 * [Intel Intrinsics Guide](https://software.intel.com/sites/landingpage/IntrinsicsGuide/)
 * [Arm Neon Intrinsics Reference](https://developer.arm.com/architectures/instruction-sets/simd-isas/neon/intrinsics)
+* [NEON Programmer's Guide](https://static.docs.arm.com/den0018/a/DEN0018A_neon_programmers_guide_en.pdf)
```

Comment:
This material was a bit old, and there was no A64 specific instruction.
How about this? https://developer.arm.com/architectures/instruction-sets/simd-isas/neon/neon-programmers-guide-for-armv8-a

Reply:
Let me study this document!

Reply:
I prefer to preserve both documents, since the old version describes permutation with figures which is easier to understand.

Reply:
I prefer to preserve both documents, since the older one mentioned some arithmetic operations which are not mentioned in the newer one.

Reply:
hsub is one of the examples which has not been listed in the newer document

Reply:
List "Neon Programmer's Guide for Armv8-A" before "NEON Programmer’s Guide Version: 1.0"